config/i18n.json

```
@@ -583,6 +583,25 @@
     "MissingClassDescription": "The build you're trying to view is for a character class you don't have.",
     "NewEmptyLoadout": "Create Empty Loadout",
     "NoBuildsFound": "No builds found. Try expanding your filtering criteria.",
+    "NoBuildsFoundExplainer": {
+      "Header": "No builds found. Here are possible reasons DIM couldn't find any sets:",
+      "AlwaysInvalidMods": "These mods don't fit into any of your items:",
+      "RemoveMods": "Remove these mods",
+      "MaybeRemoveMods": "Consider removing mods",
+      "AssumptionsRestricted": "DIM is restricted in its allowed assumptions about armor energy",
+      "AssumeElementChange": "Assume armor will change energy type",
```

```suggestion
      "AssumeElementChange": "Allow DIM to recommend armor energy element changes",
```
outside the necessarily brief toggle labels, "assume" is a strange word in some ways.
in general i would love to say things more like "Allow DIM to assume you'll masterwork armor" to represent to the user we expect something of them. but kind of not into more second person than necessary. "you" seems like a complicated pronoun to internationalize and keep consistent.

kind of into the idea of "planning" or "recommending" being a more accurate overall idea of what the options do.
either way, considering these are meant as help for possibly frustrated or novice users, would love to see them be a little clearer/more verbose